src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="search-bar"
+      data-qa="big"
+    >
+      <label class="search-bar_lab">
+        <input
+          type="text"
+          name="search-big"
+          class="search-bar_input search-bar_input__big"
```

Comment:
```suggestion
          name="searchBig"
          class="search-bar_input search-bar_input--big"
```
![image](https://user-images.githubusercontent.com/96409563/209793258-61be603a-7d78-4dd1-b49b-7d75d681c2cb.png)
![image](https://user-images.githubusercontent.com/96409563/209793294-90f02260-5d7c-47dd-ae67-3cfb742518e3.png)